python/dgl/transforms/module.py

```diff
@@ -368,39 +368,75 @@ class LaplacianPE(BaseTransform):
     <https://arxiv.org/abs/2003.00982>`__
 
     This module only works for homogeneous bidirected graphs.
-
+    
     Parameters
     ----------
     k : int
-        Number of smallest non-trivial eigenvectors to use for positional encoding
-        (smaller than the number of nodes).
+        Number of smallest non-trivial eigenvectors to use for positional encoding.
     feat_name : str, optional
         Name to store the computed positional encodings in ndata.
-
+    padding : bool
+        If False, raise an exception when k>=n.
+        Otherwise, add zero paddings in the end when k>=n.
+        Default: False.
+        n is the number of nodes in the given graph.
+    return_eigval : bool
+        If True, return laplacian eigenvalues together with eigenvectors.
+        Otherwise, return laplacian eigenvectors only. 
+        Default: False.
+    
     Example
     -------
-
     >>> import dgl
     >>> from dgl import LaplacianPE
-
-    >>> transform = LaplacianPE(k=3)
+    >>> transform1 = LaplacianPE(k=3)
+    >>> transform2 = LaplacianPE(k=5, padding=True)
+    >>> transform3 = LaplacianPE(k=5, feat_name='eigvec', eigval_name='eigval', padding=True, return_eigval=True)
     >>> g = dgl.rand_graph(5, 10)
-    >>> g = transform(g)
-    >>> print(g.ndata['PE'])
-    tensor([[ 0.0000, -0.3646,  0.3646],
-            [ 0.0000,  0.2825, -0.2825],
-            [ 1.0000, -0.6315,  0.6315],
-            [ 0.0000,  0.3739, -0.3739],
-            [ 0.0000, -0.1663,  0.1663]])
+    >>> g1 = transform1(g)
+    >>> print(g1.ndata['PE'])
+    tensor([[-0.0697, -0.0697,  0.0131],
+            [ 0.0953,  0.0953, -0.3502],
+            [ 0.3501,  0.3501,  0.5598],
+            [ 0.3929,  0.3929, -0.2883],
+            [-0.5751, -0.5751, -0.2018]])
+    >>> g2 = transform2(g)
+    >>> print(g2.ndata['PE'])
+    tensor([[-0.0697,  0.0697,  0.0131,  0.0131,  0.0000],
+            [ 0.0953, -0.0953, -0.3502, -0.3502,  0.0000],
+            [ 0.3501, -0.3501,  0.5598,  0.5598,  0.0000],
+            [ 0.3929, -0.3929, -0.2883, -0.2883,  0.0000],
+            [-0.5751,  0.5751, -0.2018, -0.2018,  0.0000]])
+    >>> g3 = transform3(g)
+    >>> print(g3.ndata['eigval'])
+    tensor([[0.9158, 0.9158, 1.4175, 1.4175,    nan],
+            [0.9158, 0.9158, 1.4175, 1.4175,    nan],
+            [0.9158, 0.9158, 1.4175, 1.4175,    nan],
+            [0.9158, 0.9158, 1.4175, 1.4175,    nan],
+            [0.9158, 0.9158, 1.4175, 1.4175,    nan]])
+    >>> print(g3.ndata['eigvec'])
+    tensor([[ 0.0697, -0.0697,  0.0131, -0.0131,  0.0000],
+            [-0.0953,  0.0953, -0.3502,  0.3502,  0.0000],
+            [-0.3501,  0.3501,  0.5598, -0.5598,  0.0000],
+            [-0.3929,  0.3929, -0.2883,  0.2883,  0.0000],
+            [ 0.5751, -0.5751, -0.2018,  0.2018,  0.0000]])
     """
-    def __init__(self, k, feat_name='PE'):
+    def __init__(self, k, feat_name='PE', eigval_name=None, padding=False, return_eigval=False):
```

Comment:
I think `return_eigval` is unnecessary here if we have the argument `eigval_name`. Just omit eigenvalues if `eigval_name` is None. Besides, please update the documentation part accordingly.